packages/angular-cli/upgrade/version.ts

```diff
@@ -94,7 +94,7 @@ export class Version {
         workflow, a faster build and smaller bundles.
         
         To get more info, including a step-by-step guide to upgrade the CLI, follow this link:
-        https://github.com/angular/angular-cli/wiki/Upgrading-from-Beta.10-to-Beta.12
+        https://github.com/angular/angular-cli/wiki/Upgrading-from-Beta.10-to-Beta.14
```

Comment:
If we sometimes change links on the wiki, then this should use a perma-link redirection service